api/files/serializers.py

```diff
@@ -232,6 +232,8 @@ def update(self, instance, validated_data):
 
         for attr, value in validated_data.items():
             if attr == 'checkout':
+                if value == '':
```

Comment:
This seems like it shouldn't be done here, and might not need doing at all. 

In `def check_in_or_out`, `action = NodeLog.CHECKED_OUT if <value> else NodeLog.CHECKED_IN` should result in `CHECKED_IN` for falsey `value`s, including `''` and `None`